### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by UnKn0wN-Binary**
Updated the Readme to better detail how to install the AWS CLI and CDK so users can go from A-Z with the base instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,66 @@ This is a CDK Project for spinning up a [Valheim](https://store.steampowered.com
 Uses [valheim-server-docker](https://github.com/lloesche/valheim-server-docker) - thanks to lloesche for putting it together!
 
 ## Table of Contents
-- [Installation](#installationdeployment)
-- [Configuration](#configuration)
-- [Cost Information](#solution-cost-information)
-- [FAQ](#common-problemsfaq)
-  * [Find your server IP](#how-do-i-find-the-ip-of-my-server)
-
+- [valheim-ecs-fargate-cdk](#valheim-ecs-fargate-cdk)
```
Comment (rileydakota) [inline README.md:10]: Having the title in ToC seems redundant

File: README.md [hunk 2]
```diff
@@ -7,12 +7,66 @@ This is a CDK Project for spinning up a [Valheim](https://store.steampowered.com
...
+
+## Fresh Installation
+1. To follow [Installation/Deployment](#installationdeployment), you will first need to install the AWS CLI alongside the AWS CDK. The below will show you how to do so. 
+2. First we need to install Node.js which also install NPM. You can download it [here](https://nodejs.org/en/)
+3. Once Node is installed you can then install the AWS CLI from [here](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+   1. Make sure you install the AWS CLI here "C:\Program Files\Amazon\AWSCLIV2"
+   2. For other Operating Systems, it should be the same instructions below, but you'll need to research the documentation corresponding to the OS Install.
+   3. Next let's open up our AWS Console, where we will create a user in IAM that can create and interact with the CDK and CLI
+   4. Go to this [link](https://console.aws.amazon.com/iam/home#/home)
+   5. Click "Users" on the left hand side
```
Comment (rileydakota) [inline README.md]: Can we link to an existing set of instructions for this? Ala: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds

Comment (rileydakota) [reply README.md]: Or can we move these to a separate README page with a link?

**Review by rileydakota — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (UnKn0wN-Binary) [reply README.md:10]: Adding in and re-pulling

File: README.md (hunk 2 above)
Comment (UnKn0wN-Binary) [reply README.md]: Added. re-pulling

**Review by madtimber — APPROVED** [review]

**Comment by UnKn0wN-Binary** [comment]
Going to write this with other instructions to use AWS CloudShell.

**Pull request closed without merging**